### PR TITLE
Project accessibility

### PR DESCRIPTION
**Pull request by GintsM**
- Page title: Changed page title to make it more unique, and easier to recognize.
- Image text alternatives: Changed into places, for background removed, since that is for styling and made more readable 'alt' 
- Text headings: No issues found
- Color contrast: improved contrast with text color for all buttons. 
- Resize: No issues found
- Interaction: Added :focus-visible for keyboard users. 
- Moving content: No issues found
- Multimedia: No issues found
- Basic structure of the page: No issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,11 +10,13 @@
...
     gtag('config', 'G-4BSWDT0W2V');
     </script>
+   
+
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
-    <title>Portfolio</title>
+    <title> Somdotta Sarkar's Portfolio</title>
```
Comment (Somdotta07) [inline index.html:19]: - Changed the Title to make it more unique.

File: index.html [hunk 2]
```diff
@@ -44,7 +46,7 @@ <h4>I'm a Software Developer</h4>
                  If you like what you see and have <br>a project you need coded, 
                 don’t hestiate to contact me.</p>
             </div>
-            <img class="background" src="media/background.png" alt="Background">  
+            <img class="background" src="media/background.png" alt="">
```
Comment (Somdotta07) [inline index.html:49]: - Removed unnecessary image description

File: index.html [hunk 3]
```diff
@@ -59,11 +61,11 @@ <h4>I'm a Software Developer</h4>
               
         </section>
     
-    <section class="project-container">
+    <section id="portfolio" class="project-container">
         <h2 class="project-heading">My Recent Works</h2>
         <hr>
         <div class="project">
-        <img  class="img"  src="media/image.png"  alt="image">
+        <img  class="img"  src="media/image.png"  alt="Project Screenshot">
```
Comment (Somdotta07) [inline index.html:68]: - Changed to more meaningful alt

File: style.css [hunk 4]
```diff
@@ -202,7 +206,7 @@ hr {
 .btn {
   display: flex;
   background: #ff6b00;
-  color: #fff;
+  color: #000;
```
Comment (Somdotta07) [inline style.css:209]: - Changed font color to improve the contrast

File: style.css [hunk 5]
```diff
@@ -119,6 +119,10 @@ p {
   margin-bottom: 3px;
 }
 
+:focus-visible {
+  border: 4px dotted black;
+}
```
Comment (Somdotta07) [inline style.css:124]: - Added focus to give keyboard interaction

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request closed without merging**